aim/storage/treeview.py

```diff
@@ -89,6 +89,13 @@ def __setitem__(
     ):
         self.set(path, value, strict=True)
 
+    @abstractmethod
+    def keys_eager(
```

Comment:
please add a TODO to revisit this approach.
do not want to do it now but after @mahnerak finishes the cython bindings implementation to avoid conflicts as much as possible.

Reply:
sure